zh/advanced-statefulset.md

```diff
@@ -0,0 +1,156 @@
+
+## 开启
+
+1. 载入 Advanced StatefulSet 的 CRD 文件：
+
+    Kubernetes 1.16 之前版本：
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/manifests/advanced-statefulset-crd.v1beta1.yaml
+    ```
+
+    Kubernetes 1.16 之后:
+
+    {{< copyable "shell-regular" >}}
+
+    ```
+    kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/manifests/advanced-statefulset-crd.v1.yaml
+    ```
+
+2. 在 TiDB Operator chart 的 values 中启用 `AdvancedStatefulSet` 特性：
+
+    {{< copyable "shell-regular" >}}
+
+    ```yaml
+    features:
+    - AdvancedStatefulSet=true
+    advancedStatefulset:
+      create: true
+    ```
+
+    然后升级 TiDB Operator ，具体可参考[升级 TiDB Operator 文档]((upgrade-TiDB Operator.md))。
+
+> **注意：**
+>
+> TiDB Operator 通过开启 `AdvancedStatefulSet` 特性，会将当前 `StatefulSet`
+> 对象转换成 `AdvancedStatefulSet 对象。但并不支持关闭 `AdvancedStatefulSet`
+> 特性后，自动从 `AdvancedStatefulSet` 转换为 Kubernetes 内置的 `StatefulSet`
+> 对象。
+
+## 使用
+
+### 通过 kubectl 查看 AdvancedStatefulSet 对象
+
+`AdvancedStatefulSet` 数据格式与 `StatefulSet` 完全一致，但以 CRD
+方式实现，别名为 `asts` ，可通过一下方法查看命令空间下的对象。
+
+{{< copyable "shell-regular" >}}
+
+```shell
+kubectl get -n <namespace> asts
+```
+
+### 操作 TidbCluster 对象指定 pod 进行缩容
+
+使用增强型 StatefulSet 时，在对 TidbCluster 进行缩容时，除了减少副本数，可同时通过配置 annotations 指定对 pd, tidb 或 tikv 组件下任意一个 pod 进行缩容。
+
+比如：
+
+{{< copyable "" >}}
+
+```yaml
+apiVersion: pingcap.com/v1alpha1
+kind: TidbCluster
+metadata:
+  name: asts
+spec:
+  version: v3.0.12
+  timezone: UTC 
+  pvReclaimPolicy: Delete
+  pd: 
+    baseImage: pingcap/pd
+    replicas: 3
+    requests:
+      storage: "1Gi"
+    config: {}
+  tikv:
+    baseImage: pingcap/tikv
+    replicas: 4
+    requests:
+      storage: "1Gi"
+    config: {}
+  tidb:
+    baseImage: pingcap/tidb
+    replicas: 2
+    service:
+      type: ClusterIP
+    config: {}
+```
+
+会部署 4 个 tikv 实例，分别为 basic-tikv-0, basic-tikv-1, ..., basic-tikv-3。若想缩容掉 basic-tikv-1 需要在修改 `spec.tikv.replicas` 为 3 同时配置以下 annotations:
+
+{{< copyable "" >}}
+
+```yaml
+metadata:
+  annotations:
+   	tikv.tidb.pingcap.com/delete-slots: '[1]'
+```
+
+完整例子如下：
+
+{{< copyable "" >}}
+
+```yaml
+apiVersion: pingcap.com/v1alpha1
+kind: TidbCluster
+metadata:
+  annotations:
+   	tikv.tidb.pingcap.com/delete-slots: '[1]'
+  name: asts
+spec:
+  version: v3.0.12
+  timezone: UTC 
+  pvReclaimPolicy: Delete
+  pd: 
+    baseImage: pingcap/pd
+    replicas: 3
+    requests:
+      storage: "1Gi"
+    config: {}
+  tikv:
+    baseImage: pingcap/tikv
+    replicas: 3
+    requests:
+      storage: "1Gi"
+    config: {}
+  tidb:
+    baseImage: pingcap/tidb
+    replicas: 2
+    service:
+      type: ClusterIP
+    config: {}
+```
+
+支持的 annotations 为：
+
+- `pd.tidb.pingcap.com/delete-slots`: 指定 pd 组件需要删除的 pod 序号。
+- `tidb.tidb.pingcap.com/delete-slots`: 指定 tidb 组件需要删除的 pod 序号
+- `tikv.tidb.pingcap.com/delete-slots`: 指定 tikv 组件需要删除的 pod 序号
```

Comment:
Please use Chinese colons when there are both Chinese and English words in the text.
```suggestion
- `pd.tidb.pingcap.com/delete-slots`：指定 PD 组件需要删除的 Pod 序号。
- `tidb.tidb.pingcap.com/delete-slots`：指定 TiDB 组件需要删除的 Pod 序号。
- `tikv.tidb.pingcap.com/delete-slots`：指定 TiKV 组件需要删除的 Pod 序号。
```